tempest/api/workloadmgr/snapshot/test_snapshot_mount.py

```diff
@@ -530,7 +540,7 @@ def test_7_snapshot_mount_unmount_full_imagebooted_fvm_cli(self):
             else:
                 reporting.add_test_step(
                     "Snapshot unmount of full snapshot", tvaultconf.FAIL)
```

Comment:
In case of failure we are simply marking this test case to FAIL and cascading down and executing remaining part. Is it expected or we need to raise an exception and come out of this method ?